src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -222,7 +225,7 @@ public partial class SslStream : System.Net.Security.AuthenticatedStream
         public void Write(byte[] buffer) { }
         public override void Write(byte[] buffer, int offset, int count) { }
         public override System.Threading.Tasks.Task WriteAsync(byte[] buffer, int offset, int count, System.Threading.CancellationToken cancellationToken) { throw null; }
-        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public override System.Threading.Tasks.ValueTask WriteAsync(System.ReadOnlyMemory<byte> buffer, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
This shouldn't be removed; it's both a breaking change and an inconvience.  If that's being done by the ref generator because it's missing from the implementation, we should instead add it into the implementation.

Reply:
Yeah I just noticed that it (ref generator) did some unexpected changes (there's also a case of `~SslStream` destructor).
I'll clean it up.

Reply:
Fixed